sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -689,14 +616,8 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
     }
     assert(filters.nonEmpty) // filter over aggregate not pushed down
     checkAggregateRemoved(df)
```

Comment:
Should this be `checkAggregateRemoved(query)`? Not related to this PR, though.

Reply:
I see.